// [[Reason: Invalid computed property can only be detected by TSC]] ////

//// [tests/cases/compiler/indexWithoutParamType2.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,8 +1,9 @@
 
 
 //// [indexWithoutParamType2.d.ts]
 declare class C {
+    [x]: string;
 }
 
 /// [Errors] ////
 
